libs/abcvlib/download_models.gradle

```diff
@@ -1,15 +1,20 @@
-project.ext.ASSET_DIR = projectDir.toString() + '/src/main/assets'
-project.ext.MODEL_FILE_URL = 'https://github.com/oist/smartphone-robot-object-detection/releases/download/0.1.1/model.tflite'
+def ASSET_DIR = "${projectDir}/src/main/assets"
+def MODEL_FILES = [
+        [url: 'https://github.com/oist/smartphone-robot-object-detection/releases/download/0.1.1/model.tflite', name: 'model.tflite'],
+        [url: 'https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/android/lite-model_efficientdet_lite0_detection_metadata_1.tflite', name: 'efficientdet-lite0.tflite'],
+        [url: 'https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/android/lite-model_efficientdet_lite1_detection_metadata_1.tflite', name: 'efficientdet-lite1.tflite']
+]
 
-task downloadModelFileRobot {
-    // Only download if the file doesn't exist
-    def modelFile = file("${project.ext.ASSET_DIR}/model.tflite")
-    if (!modelFile.exists()) {
-        exec {
-            commandLine 'wget', '-N', '-P', project.ext.ASSET_DIR, project.ext.MODEL_FILE_URL
+tasks.register('downloadModelFiles') {
+    doLast {
+        MODEL_FILES.each { file ->
+            download.run {
+                src file.url
+                dest "${ASSET_DIR}/${file.name}"
+                overwrite false
+            }
```

Comment:
I'd prefer if you keep these kinds of fundamental changes (change that affect more than just this example app) in a separate PR. Otherwise this will be much harder to track down if this causes issues later on.

Reply:
you're right at least I should have did this in a separate commit